neqo-transport/src/cc/mod.rs

```diff
@@ -49,6 +49,7 @@ pub trait CongestionControl: Display + Debug {
     fn on_packet_sent(&mut self, pkt: &SentPacket);
 }
 
+#[derive(Copy, Clone)]
```

Comment:
This is good, but it's confounding the review of this PR.

Reply:
This is need for the PR to compile.